src/RwaUrn.sol

```diff
@@ -0,0 +1,305 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.6.8 <0.7.0;
```

Comment:
`pragma solidity >=0.5.12` link in [this file](https://github.com/makerdao/MIP21-RWA-Example/blob/fce06885ff89d10bf630710d4f6089c5bba94b4d/src/RwaLiquidationOracle.sol) is incorrect. Those contracts won't compile against solidity 0.7.x for example.

Other files from the RWA example repo have the versions fixed to `0.5.12`, however PE recommended we to use `0.6.12` for new contracts. Since this repo contains new contracts, we went for that version. `dapp.tools` doesn't allow for multi-version projects, so this required us to upgrade the existing contracts from the example repo.

I'm explicitly requiring `>=0.6.8` because it's the version that first introduced `type(uint256).max` as a replacement for `uint256(-1)`.

We are using `0.6.12` anyways, but if someone else ever tries to reuse this code for whatever reason, they can know for sure against which versions it will compile.